src/main.ts

```diff
@@ -64,6 +64,12 @@ export default class PeriodicNotesPlugin extends Plugin {
   }
 
   async onload(): Promise<void> {
+    addIcon("calendar-day", calendarDayIcon);
```

Comment:
Why did you move these?

Reply:
I mentioned moving them to _before_ `onLayoutReady()` but this was an oversight, I'm not entirely sure which process was blocking rendering but having them at the top meant the ribbon icons loaded instantaneously rather than popping in. The gifs highlighting this are in the description.  👍